codalab/model/bundle_model.py

```diff
@@ -1726,6 +1726,41 @@ def get_user(self, user_id=None, username=None, check_active=True):
         if result:
             return result[0]
         return None
+    
+    def search_users(self, user_id, keywords):
+        clauses = []
+        limit = 5
+        for keyword in keywords:
+            if '.limit' not in keyword:
+                clause = or_(
+                    cl_user.c.user_name.like('%' + keyword + '%'),
+                    cl_user.c.first_name.like('%' + keyword + '%'),
+                    cl_user.c.last_name.like('%' + keyword + '%'),
+                )
+                clauses.append(clause)
+        clause = and_(*clauses)
+        cols_to_select = [
+            cl_user.c.user_id,
+            cl_user.c.user_name,
+            cl_user.c.email,
+            cl_user.c.last_login,
+            cl_user.c.is_active,
+            cl_user.c.first_name,
+            cl_user.c.last_name,
+            cl_user.c.affiliation,
+            cl_user.c.time_quota,
+            cl_user.c.time_used,
+            cl_user.c.disk_quota,
+            cl_user.c.disk_used,
+            cl_user.c.disk_quota,
+        ]
```

Comment:
Are there fields we would want to include/exclude?

Reply:
Don't include sensitive information (e.g., email)!  I'd only include information that's going to show up on the webpage.  Err on the side of including less.